bin/healthcheck.js

```diff
@@ -1,24 +1,24 @@
-const http = require("http")
+const http = require('http');
 
 const options = {
-  host: "localhost",
-  port: "9999",
-  path: "/health",
-  timeout: 2000
-}
+  host: 'localhost',
+  port: '9999',
+  path: '/health',
+  timeout: 2000,
+};
 
-const request = http.request(options, (res) => {
-  console.log(`STATUS: ${res.statusCode}`)
+const request = http.request(options, res => {
```

Comment:
this change seems more confusing in this particular case, but that's just me

Reply:
This is a result of the prettier setting: `"arrowParens": "avoid"`

I totally see what you're saying, unfortunately there's not a clean way to configure it for unique situations like this.